test/ExtendedXmlSerializerTest/TestObject/TestClassCollection.cs

```diff
@@ -0,0 +1,28 @@
+﻿// MIT License
+// 
+// Copyright (c) 2016 Wojciech Nagórski
```

Comment:
I can add:

Contributions:
- Wojciech Nagórski - original version
- Mike-EEE - many improvements

It will be ok?